crates/sui-config/src/genesis.rs

```diff
@@ -286,6 +300,14 @@ pub struct GenesisChainParameters {
     /// protocol version that the chain starts at.
     #[serde(default = "ProtocolVersion::max")]
     pub protocol_version: ProtocolVersion,
+
+    #[serde(default = "GenesisChainParameters::default_allow_insertion_of_extra_objects")]
+    pub allow_insertion_of_extra_objects: bool,
```

Comment:
What is this argument for? i.e. under what condition is it true?

Reply:
This is true for a lot of the networks we've spun up and in tests. But i wanted to add this flag to be able to ensure via tooling that we didn't insert any arbitrary objects when doing a real genesis ceremony